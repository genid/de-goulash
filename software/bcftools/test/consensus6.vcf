##fileformat=VCFv4.2
##reference=consensus6.fa
##contig=<ID=NW_019171628.1,length=354362>
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	sample
NW_019171628.1	64334	.	A	AAG	159	.	.	GT	1/1
NW_019171628.1	64435	.	CACACACACA	C	170	.	.	GT	1/1
NW_019171628.1	64443	.	C	CACTCAG	72	.	.	GT	1/1
