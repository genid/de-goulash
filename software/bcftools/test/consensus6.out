>NW_019171628.1:64330-64531
GAGAAAGAGAGAGAGAGAGCATGCTTCCAAACCCTTTGTTCTTTTTCTTCTGAAGAGAAG
GTGACCTGGTTTTGAAACACTGGCTACAGCAAATCACTCAGAACACACACACACACACAC
ACACACACACACACACACACACTCTCAGAACACACACACACACACGCACTTACATTTAGC
CTTGGTGTTACTAT
